dll/win32/msgina/resource.h

```diff
@@ -99,9 +99,11 @@
 #define IDS_SHUTDOWN_RESTART          50002
 #define IDS_SHUTDOWN_SLEEP            50003
 #define IDS_SHUTDOWN_HIBERNATE        50004
+#define IDS_SHUTDOWN_NT_MODE          50005
 /* Shut down descriptions */
-#define IDS_SHUTDOWN_SHUTDOWN_DESC    50005
-#define IDS_SHUTDOWN_LOGOFF_DESC      50006
-#define IDS_SHUTDOWN_RESTART_DESC     50007
-#define IDS_SHUTDOWN_SLEEP_DESC       50008
-#define IDS_SHUTDOWN_HIBERNATE_DESC   50009
+#define IDS_SHUTDOWN_SHUTDOWN_DESC    50006
```

Comment:
Do these resource ids match up with Windows? If so, you'd want to avoid moving them around and instead add your new one at the end.

Reply:
Reverted IDS values to originals and moved the new entries last. Sorry about that.